src/CabanaPD_ForceModels.hpp

```diff
@@ -23,9 +25,54 @@ struct BaseForceModel
         : delta( _delta ){};
 
     BaseForceModel( const BaseForceModel& model ) { delta = model.delta; }
+
+    // No-op for temperature.
```

Comment:
What does it mean "No-op for temperature."?

Reply:
Does nothing